Yafc.Model/Model/RecipeParameters.cs

```diff
@@ -120,7 +121,7 @@ public static RecipeParameters CalculateParameters(RecipeRow row) {
                     var fluid = recipe.ingredients[0].goods.fluid;
                     if (fluid != null) {
                         float inputTemperature = fluid.temperature;
-                        foreach (Fluid variant in fluid.variants ?? []) {
```

Comment:
Just for better understanding what the problem was and how that line solves it, could you please explain it to us?

Reply:
`row.variants` contains the variants selected for the recipe row (usually fluid ingredient temperatures, but sometimes an accumulator), while `fluid.variants` contains all possible temperatures for that fluid. This change restores the behavior of searching the temperatures selected for this row, instead of the possible temperatures for the input fluid.

Reply:
Thank you for the explanation!